### PR TITLE
Feature/ground removal

### PR DESCRIPTION
**Pull request by Davide64-dev**

### CODE REVIEW

**Comment by marhcouto** [comment]
Task linked: [CU-8693hqq1d First implementation of the ground removal](https://app.clickup.com/t/8693hqq1d)

**Review by marhcouto — CHANGES_REQUESTED** [review]
All code seems great for now. Some notes:
- There are not instructions for installation of pcl, not sure if you know that or if they are necessary, it failed to find the library on my PC
- Changed to draft because I though this was WIP, but correct it if I'm wrong
- Cpplint is not passing, check that out

File: src/perception/src/ground_removal/ransac.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+void RANSAC::groundRemoval(const pcl::PointCloud<pcl::PointXYZI>::Ptr point_cloud, pcl::PointCloud<pcl::PointXYZI> &ret) const{
+
+    pcl::ModelCoefficients::Ptr coefficients (new pcl::ModelCoefficients);
+    pcl::PointIndices::Ptr inliers_indices (new pcl::PointIndices);
+
+    pcl::SACSegmentation<pcl::PointXYZI> seg;
+
+    seg.setOptimizeCoefficients (true);
```
Comment (marhcouto) [inline src/perception/src/ground_removal/ransac.cpp:20]: Add some comments indicating what is being defined in these lines very briefly

File: src/perception/test/ransac_test.cpp [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    ground_removal->groundRemoval(pcl_cloud_3_points, ground_removed_cloud);
+
+    ASSERT_EQ(ground_removed_cloud.points.size(), 3);
+}
+
+/**
+ * @brief Test Scenario: Point cloud with 0 points - Must return a point with 0 points also.
+ * 
+ */
+TEST_F(RANSACTest, Test10) {
```
Comment (marhcouto) [inline src/perception/test/ransac_test.cpp:162]: Tests are enough to check functionality, I'll just leave the suggestion to change the name of the tests to something more alusive of the tests itself, but as it is already in the comments, I leave it to you

**Comment by marhcouto** [comment]
Task linked: [CU-8693htmxn Improve ground removal implementation](https://app.clickup.com/t/8693htmxn)

**Review by marhcouto — CHANGES_REQUESTED** [review]
Review in ClickUp

**Review by marhcouto — APPROVED** [review]

**Review by BrunoDias201907828 — APPROVED** [review]

**Pull request merged**